### PR TITLE
docs: Improve documentation around session management

### PR DESCRIPTION
**Pull request by samueldr**
>[!NOTE]
>
> It is expected that end-users have enough familiarity to draw the rest of the NixOS shaped owl at this moment in time. Otherwise their experience will be harder anyway from many other details.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
 
 To use the Steam Deck UI, set `jovian.steam.enable = true;` in your configuration.
-This will only enable the Steam Deck UI tooling. To enable "desktop" steam, enable the usual NixOS options.
-Then you can start the UI using one of the following methods:
 
-- (**preferred**) Use `jovian.steam.autoStart = true;` to auto-start at login, and enabling use of *Switch to Desktop* option.
-- Select "Gaming Mode" in the Display Manager or run `start-gamescope-session` in a VT.
-- Launch `gamescope-session` within an existing desktop session. This will run [gamescope](https://github.com/Plagman/gamescope) in nested mode which results in higher latency.
+This will only enable the Steam Deck UI tooling.
+**To enable "desktop" steam**, enable the usual NixOS options.
```
Comment (K900) [inline docs/configuration.md]: "Steam", capitalized.

File: docs/configuration.md [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
+Select the *Gaming Mode* sesssion in your Display Manager, or run `start-gamescope-session` in a VT.
+
+The *Switch to Desktop* option will not work as intended, instead it will close Steam.
+
+
+#### As a *nested* window
+
+Run `gamescope-session` within an existing desktop session.
+
+This will run [gamescope](https://github.com/Plagman/gamescope) in nested mode which may result in higher latency.
```
Comment (K900) [inline docs/configuration.md]: https://github.com/ValveSoftware/gamescope is the correct upstream

File: docs/steam.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ To manually start **Gaming Mode** (also known as the **Steam Deck UI**), there a
 
 ## Start On Boot
 
-To automatically launch Gaming Mode on boot and enable desktop switching from the power menu:
+The following configuration snippet configures the system to automatically launch Gaming Mode on boot, and enables desktop switching to Plasma 5.
```
Comment (K900) [inline docs/steam.md]: Maybe we should update this to Plasma 6.

Comment (samueldr) [reply docs/steam.md]: Can you share the exact config you'd ~~support~~ recommend for this? Just so I'm off the hook :).

File: docs/configuration.md (hunk 2 above)
Comment (samueldr) [reply docs/configuration.md]: Thanks, updated.

File: docs/configuration.md (hunk 1 above)
Comment (samueldr) [reply docs/configuration.md]: Good catch, thanks.

File: docs/steam.md (hunk 3 above)
Comment (K900) [reply docs/steam.md]: Just replace `services.xserver.desktopManager.plasma5.enable = true` with `services.desktopManager.plasma6.enable = true`

File: docs/steam.md [hunk 4]
```diff
@@ -17,11 +17,11 @@ To manually start **Gaming Mode** (also known as the **Steam Deck UI**), there a
 
 ## Start On Boot
 
-The following configuration snippet configures the system to automatically launch Gaming Mode on boot, and enables desktop switching to Plasma 5.
+The following configuration snippet configures the system to automatically launch Gaming Mode on boot, and enables desktop switching to Plasma 6.
 
 ```nix
 {
-  services.xserver.desktopManager.plasma5.enable = true;
+  services.desktopManager.plasma6.enable = true
```
Comment (K900) [inline docs/steam.md]: ;

Comment (samueldr) [reply docs/steam.md]: :facepalm:

**Review by K900 — APPROVED** [review]

**Comment by samueldr** [comment]
*sending it*

**Pull request merged**